presidio-anonymizer/presidio_anonymizer/entities/engine/decrypt_config.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from presidio_anonymizer.entities.engine import OperatorMetadata
+from presidio_anonymizer.operators import Decrypt
+from presidio_anonymizer.operators import OperatorType
+from presidio_anonymizer.services.validators import validate_parameter_exists
+
+
+class DecryptConfig(OperatorMetadata):
+    """Decrypt configuration for each text entity of the decryption."""
+
+    def __init__(self, key: str):
+        """
+        Create AnonymizerConfig entity.
+
+        :param anonymizer_name: the anonymizer name string - represents the class
+        of the anonymizer in lower case letters. e.g.: redact
+        :param params: the parameters to use in the selected anonymizer class
+        """
+        self.logger = logging.getLogger("presidio-anonymizer")
+        validate_parameter_exists(key, "config", "key")
+        params = {Decrypt.KEY: key}
```

Comment:
I brought it up in the previous comment you made... I am waiting for Nava's reply.

Reply:
I think we will have a problem here, Deanoynmize currently gets a key do decrypt. We don't have data about other types of decryption and what params they are going to need and use.
We are going to do something in preparation of without knowing how it is going to be or look.

Reply:
The idea is to have the same structure. By having `params` on the operators we are going very generic. Why not have the same structure here?

Reply:
My main concern is not about whether we will have more deanonymizers or not in the future. It's about he simplicity and readability of the code. It takes quite a while to understand what's going on and this will prevent users from extending it.